### ISSUE TITLE
No dll in the newest release

### ISSUE THREAD

**Issue by brian6932** [ImageOptim/gifski#119]
The last dll was on the 0.8.5 release, 10.1 does not have one

**Comment by kornelski** [comment]
I don't recall publishing a DLL.

If you want one:

1. Install Rust https://www.rust-lang.org/tools/install and restart your terminal to see new `PATH`

2. ```sh
   git clone https://github.com/ImageOptim/gifski
   cd gifski
   cargo build --lib --release
   ```

3. The DLL will be in `target\release\gifski.dll`

**Comment by DumbJoe** [comment]
> The last dll was on the 0.8.5 release, 

No it didn't, check: https://github.com/ImageOptim/gifski/releases/download/0.8.5/gifski-0.8.5.tar.xz

Also, why do you even need a dll for anyways if one file is good enough?

**Comment by DumbJoe** [comment]
> The last dll was on the 0.8.5 release, 

No it didn't, check: https://github.com/ImageOptim/gifski/releases/download/0.8.5/gifski-0.8.5.tar.xz

Also, why do you even need a dll for anyways if one file is good enough?

**Comment by DumbJoe** [comment]
> The last dll was on the 0.8.5 release, 

No it didn't, check: https://github.com/ImageOptim/gifski/releases/download/0.8.5/gifski-0.8.5.tar.xz

Also, why do you even need a dll for anyways if one file is good enough?

**Comment by DumbJoe** [comment]
> The last dll was on the 0.8.5 release, 

No it didn't, check: https://github.com/ImageOptim/gifski/releases/download/0.8.5/gifski-0.8.5.tar.xz

Also, why do you even need a dll for anyways if one file is good enough?

**Comment by DumbJoe** [comment]
> The last dll was on the 0.8.5 release, 

No it didn't, check: https://github.com/ImageOptim/gifski/releases/download/0.8.5/gifski-0.8.5.tar.xz

Also, why do you even need a dll for anyways if one file is good enough?

**Comment by DumbJoe** [comment]
> The last dll was on the 0.8.5 release, 

No it didn't, check: https://github.com/ImageOptim/gifski/releases/download/0.8.5/gifski-0.8.5.tar.xz

Also, why do you even need a dll for anyways if one file is good enough?

**Comment by DumbJoe** [comment]
> The last dll was on the 0.8.5 release, 

No it didn't, check: https://github.com/ImageOptim/gifski/releases/download/0.8.5/gifski-0.8.5.tar.xz

Also, why do you even need a dll for anyways if one file is good enough?

**Comment by DumbJoe** [comment]
> The last dll was on the 0.8.5 release, 

No it didn't, check: https://github.com/ImageOptim/gifski/releases/download/0.8.5/gifski-0.8.5.tar.xz

Also, why do you even need a dll for anyways if one file is good enough?

**Comment by DumbJoe** [comment]
> The last dll was on the 0.8.5 release, 

No it didn't, check: https://github.com/ImageOptim/gifski/releases/download/0.8.5/gifski-0.8.5.tar.xz

Also, why do you even need a dll for anyways if one file is good enough?

**Comment by DumbJoe** [comment]
> The last dll was on the 0.8.5 release, 

No it didn't, check: https://github.com/ImageOptim/gifski/releases/download/0.8.5/gifski-0.8.5.tar.xz

Also, why do you even need a dll for anyways if one file is good enough?

**Comment by DumbJoe** [comment]
> The last dll was on the 0.8.5 release, 

No it didn't, check: https://github.com/ImageOptim/gifski/releases/download/0.8.5/gifski-0.8.5.tar.xz

Also, why do you even need a dll for anyways if one file is good enough?

**Comment by DumbJoe** [comment]
> The last dll was on the 0.8.5 release, 

No it didn't, check: https://github.com/ImageOptim/gifski/releases/download/0.8.5/gifski-0.8.5.tar.xz

Also, why do you even need a dll for anyways if one file is good enough?

**Comment by DumbJoe** [comment]
> The last dll was on the 0.8.5 release, 

No it didn't, check: https://github.com/ImageOptim/gifski/releases/download/0.8.5/gifski-0.8.5.tar.xz

Also, why do you even need a dll for anyways if one file is good enough?

**Comment by DumbJoe** [comment]
> The last dll was on the 0.8.5 release, 

No it didn't, check: https://github.com/ImageOptim/gifski/releases/download/0.8.5/gifski-0.8.5.tar.xz

Also, why do you even need a dll for anyways if one file is good enough?

**Comment by DumbJoe** [comment]
> The last dll was on the 0.8.5 release, 

No it didn't, check: https://github.com/ImageOptim/gifski/releases/download/0.8.5/gifski-0.8.5.tar.xz

Also, why do you even need a dll for anyways if one file is good enough?

**Comment by DumbJoe** [comment]
> The last dll was on the 0.8.5 release, 

No it didn't, check: https://github.com/ImageOptim/gifski/releases/download/0.8.5/gifski-0.8.5.tar.xz

Also, why do you even need a dll for anyways if one file is good enough?

**Comment by DumbJoe** [comment]
> The last dll was on the 0.8.5 release, 

No it didn't, check: https://github.com/ImageOptim/gifski/releases/download/0.8.5/gifski-0.8.5.tar.xz

Also, why do you even need a dll for anyways if one file is good enough?

**Comment by brian6932** [comment]
I know I haven't replied in this thread for a while, but for reference, the reason I wanted the .dll file was because ScreenToGif exclusively uses .dll files, and the one that I was using with it was buggy and had trouble with high fps. The method that @kornelski recommended does work, although it would be pretty nice to just have it bundled in the release file imo.

**Comment by Vankog** [comment]
Is there a way to get the DLL without any self-building?

**Comment by kornelski** [comment]
No, you need to build yourself. But it shouldn't be too hard (see previous comment)
